tests/console/commands/test_remove.py

```diff
@@ -1,13 +1,161 @@
 import pytest
+import tomlkit
 
 from poetry.core.packages.package import Package
+from poetry.factory import Factory
 
 
 @pytest.fixture()
 def tester(command_tester_factory):
     return command_tester_factory("remove")
 
 
+def test_remove_without_specific_group_removes_from_all_groups(
+    tester, app, repo, command_tester_factory, installed
+):
+    """
+    Removing without specifying a group removes packages from all groups.
+    """
+    installed.add_package(Package("foo", "2.0.0"))
```

Comment:
What do you think about maybe creating constants for the strings `baz`, `foo`, `1.0.0`, and `2.0.0`, since they are repeated so often?

If you agree, you will need to change the other test files that use these same values.